src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/GetStartedPage.razor.scss

```diff
@@ -188,6 +188,11 @@
     line-height: rem(24px);
     color: $B2Color;
     background-color: $B6Color;
+    overflow: auto;
+
+    &::-webkit-scrollbar:horizontal {
+        height: rem(6px);
+    }
```

Comment:
shouldn't we move these kinds of CSS properties into a more basic file?
@maryamhdr what do you think?

Reply:
This is not the correct solution...Let have a meeting @sajadkardel